docs/components/nav_drawer.md

```diff
@@ -0,0 +1,324 @@
+<!--docs:
+title: "Material <component>"
+layout: detail
+section: components
+excerpt: "Navigation drawers provide access to destinations in your app."
+iconId:
+path: /catalog/NavigationDrawers/
+-->
+
+# Navigation drawer
+
+A [navigation drawer](https://material.io/components/navigation-drawer) provides access to destinations in your app.
+
+![Hero navigation drawer image](assets/nav_drawer/NavDrawer-hero.png)
+
+## Contents
+
+* [Using a navigation drawer](#using-a-navigation-drawer)
+* [Making navigation drawers accessible](#making-navigation-drawers-accessible)
+* [Anatomy](#anatomy)
+* [Types](#types)
+* [Standard navigation drawer](#standard-navigation-drawer)
+* [Modal navigation drawer](#modal-navigation-drawer)
+* [Bottom navigation drawer](#bottom-navigation-drawer)
+* [Theming a navigation drawer](#theming-a-navigation-drawer)
```

Comment:
Switch * to - to be consistent with other docs

Reply:
Done